src/test/Spec.hs

```diff
@@ -74,12 +73,13 @@ solidityTests = testGroup "Solidity-HUnit"
         assertBool "s2 not in solution" $ any (\case {Left ("s2", _) -> True; _ -> False;}) calls
         assertBool "s3 not in solution" $ any (\case {Left ("s3", _) -> True; _ -> False;}) calls
   ]
-  where c2   = "./examples/solidity/basic/flags.sol"
-        c3   = "./examples/solidity/basic/revert.sol"
-        --c4   = "./examples/solidity/basic/payable.sol"
-        c5   = "./examples/solidity/basic/true.sol"
-        c6   = "./examples/solidity/basic/multisender.sol"
-        cfg6 = "./examples/solidity/basic/multisender.yaml"
+  where cd   = "./examples/solidity/basic/"
```

Comment:
instead of doing `cd ++ "whatever"` here we could do `main = setCurrentDirectory "./examples/solidity/basic" >> defaultMain tests`

Reply:
Done (@japesinator)